test/com/datastax/configbuilder/operator_test.clj

```diff
@@ -36,10 +37,6 @@
       :rack-name "default"
       :config-output-directory (str @helper/temp-dir)
       :definitions-location "cass-config-definitions/resources"})
-    (loop [files (.list (.toFile @helper/temp-dir))]
-      (let [current-file (first files)
-            diff-result (sh "diff"
-                            (str @helper/temp-dir "/" current-file)
-                            (str "test/data/configs/dse-6.0.11/" current-file))]
-        (is (= 0 (:exit diff-result))
-            (str "Rendered output for " current-file " does not match expected output.\nDiff:\n" (:out diff-result)))))))
+    (is (= #{} (set/difference
+      #{"logback.xml" "cassandra-rackdc.properties" "cassandra-env.sh" "dse-env.sh" "cassandra.yaml" "dse.yaml" "10-write-prom.conf" "jvm.options"}
+      (into #{} (.list (.toFile @helper/temp-dir))))))))
```

Comment:
This test does not seem to have worked previously. The sort of testing attempted here is not well supported by the code we have for conducting golden file tests presently. Consequently, I've changed this test to just check that files are being generated rather than examining their content. We have other tests that look at how content is generated.

Reply:
That works for me.